### ISSUE TITLE
Meteor 1.3.1 causes OSX 10.11.4 to freeze

### ISSUE THREAD

**Issue by eminnett** [meteor/meteor#6772]
Attempting to install Meteor and create a project is causing my computer to freeze. The only recourse I am left with when this happens is to perform a hard reset of OSX. I am using meteor 1.3.1 on OSX 10.11.4. I have tried uninstalling and reinstalling meteor, creating the application in different places, and restarting the computer (from a non-frozen state) all leading to the same result.

These are the commands I have entered to get to frozen state:

```
$ curl https://install.meteor.com/ | sh
$ meteor create my_first_app
$ cd my_first_app/
$ meteor npm install
$ meteor
```

The output freezes when it displays `Loading plugin 'minifyStdCSS' from pa...  -` and a few moments later, the terminal freezes followed by the whole operating system.

The following is the terminal output with the prompt replaced by '$':

```
$ curl https://install.meteor.com/ | sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  6674    0  6674    0     0  11164      0 --:--:-- --:--:-- --:--:-- 11160
Downloading Meteor distribution
######################################################################## 100.0%

Meteor 1.3.1 has been installed in your home directory (~/.meteor).
Writing a launcher script to /usr/local/bin/meteor for your convenience.

To get started fast:

  $ meteor create ~/my_cool_app
  $ cd ~/my_cool_app
  $ meteor

Or see the docs at:

  docs.meteor.com

$ meteor create my_first_app
Created a new Meteor app in 'my_first_app'.   

To run your new app:                          
  cd my_first_app                             
  meteor                                      
                                              
If you are new to Meteor, try some of the learning resources here:
  https://www.meteor.com/learn                
                                              
$ cd my_first_app/
$ meteor npm install
meteor-node-stubs@0.2.3 node_modules/meteor-node-stubs
├── https-browserify@0.0.1
├── tty-browserify@0.0.0
├── path-browserify@0.0.0
├── os-browserify@0.2.1
├── constants-browserify@1.0.0
├── string_decoder@0.10.31
├── process@0.11.2
├── punycode@1.4.1
├── assert@1.3.0
├── querystring-es3@0.2.1
├── timers-browserify@1.4.2
├── domain-browser@1.1.7
├── events@1.1.0
├── stream-browserify@2.0.1 (inherits@2.0.1)
├── util@0.10.3 (inherits@2.0.1)
├── vm-browserify@0.0.4 (indexof@0.0.1)
├── console-browserify@1.1.0 (date-now@0.1.4)
├── http-browserify@1.7.0 (inherits@2.0.1, Base64@0.2.1)
├── readable-stream@2.0.6 (process-nextick-args@1.0.6, inherits@2.0.1, util-deprecate@1.0.2, isarray@1.0.0, core-util-is@1.0.2)
├── url@0.11.0 (punycode@1.3.2, querystring@0.2.0)
├── buffer@4.5.1 (isarray@1.0.0, ieee754@1.1.6, base64-js@1.1.2)
├── browserify-zlib@0.1.4 (pako@0.2.8)
└── crypto-browserify@3.11.0 (inherits@2.0.1, create-hmac@1.1.4, randombytes@2.0.3, pbkdf2@3.0.4, create-hash@1.1.2, diffie-hellman@5.0.2, create-ecdh@4.0.0, public-encrypt@4.0.0, browserify-cipher@1.0.0, browserify-sign@4.0.0)
$ meteor
[[[[[                                         
~/Desktop/Workspace/courses/coursera/responsive_web_design_specialisation/3_meteor_development/week_1/my_first_app
]]]]]

=> Started proxy.                             
=> Started MongoDB.                           
   Loading plugin `minifyStdCSS` from pa...  -
  [Restored 10 Apr 2016, 18:19:51]
```

**Comment by zaiz91** [comment]
I have exactly the same problem. Anyone have a solution for this?

**Comment by abernix** [comment]
@eminnett Can you try this again on the latest version of Meteor 1.3.2.4?

**Comment by shantanubhadoria** [comment]
Bump. Same issue on 1.3.2.4 here.

**Comment by laosb** [comment]
According to @zaiz91 , that might be related to #5691.

**Comment by abernix** [comment]
@shantanubhadoria Could you clarify your *exact* circumstances?  Because this isn't too far off from several similar issues.  Does it happen when you create a completely brand new, empty app?  And do you have a virus scanner running?  If so, which one?

**Comment by shantanubhadoria** [comment]
I have kaspersky running. Yes this happens when I create a brand new app and try to run meteor on it.
Do note that it works fine if I try to run one my un-upgraded older meteor apps.

**Comment by abernix** [comment]
@shantanubhadoria Does it go away if you temporarily disable Kaspersky?

@eminnett Are you also running anti-virus software?  If so, can you disable it, try again and report back?

Thanks!

**Comment by eminnett** [comment]
I also use Kaspersky. I'll make a note to give it another go at the weekend.

Cheers.

**Comment by eminnett** [comment]
I have just attempted to install a fresh Meteor app with and without Kaspersky protection enabled. With Kaspersky running normally, I was forced to do a hard restart after attempting to execute `$ meteor create my_first_app`. With Kaspersky protection turned off, I was able to install and run a fresh Meteor app without any problems.

Thanks for your help everybody in figuring out the cause of the problem. 

m going to go ahead and close the issue.
